lib/src/golden_test_theme.dart

```diff
@@ -29,4 +29,30 @@ class GoldenTestTheme {
 
   /// The border color used to separate scenarios in a [GoldenTestGroup].
   final Color borderColor;
+
+  @override
+  ThemeExtension<GoldenTestTheme> copyWith({
```

Comment:
Is there a benefit to making this a proper `ThemeExtension` or are we just wanting `copyWith` functionality?